packages/react-auth/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.0.17 - 2025-03-25
+
+### Updated
+
+- Upgraded @pangeacyber/vanilla-js auth client version
```

Comment:
Does the AuthConfig interface in react-auth need to also be updated?

Reply:
Yes, indeed, updated